front/src/Apps/Dashboard/Creation/bsvhu/steps/Transporter.tsx

```diff
@@ -176,13 +198,56 @@ const TransporterBsvhu = ({ errors }) => {
           key={orgId}
         />
       </div>
+      {!!orgId &&
+        !isForeign &&
+        transporter.transport.mode === TransportMode.Road &&
+        !transporter.recepisse.isExempted && (
+          <TransporterRecepisse
+            number={recepisse?.number}
+            department={recepisse?.department}
+            validityLimit={recepisse?.validityLimit}
+          />
+        )}
       <div className="fr-col-md-12 fr-mt-4w">
         <RecepisseExemption
           onChange={v => setValue(`${actor}.recepisse.isExempted`, v)}
           checked={transporter.recepisse?.isExempted}
           disabled={sealedFields.includes(`${actor}.recepisse.isExempted`)}
         />
       </div>
+      <div className="fr-grid-row">
+        <div className="fr-col-6">
+          <Select
+            label="Mode de transport"
+            disabled={sealedFields.includes(`transporter.transport.mode`)}
+            nativeSelectProps={{
+              ...register("transporter.transport.mode")
+            }}
+          >
+            <option value="ROAD">Route</option>
+            <option value="RAIL">Voie Ferroviaire</option>
+            <option value="AIR">Voie Aérienne</option>
+            <option value="RIVER">Voie Fluviale</option>
+            <option value="SEA">Voie Maritime</option>
+            <option value="OTHER">Autre</option>
```

Comment:
{Object.entries(transportModeLabels).map(([k, v]) => (
        <option value={`${k}`} key={k}>
          {v}
        </option>
      ))}

et je vois dans dashboard/constants 
/**
 * Subset of TransportMode
 * We don't want Other to be displayed in the front
 */
enum PublicTransportMode {
  Air = "AIR",
  Rail = "RAIL",
  River = "RIVER",
  Road = "ROAD",
  Sea = "SEA"
}
donc à voir si il faut afficher "autre" ou pas dans les choix

Reply:
good catch